src/modules/service-catalog/ServiceCatalogItemPage.tsx

```diff
@@ -147,27 +109,30 @@ export function ServiceCatalogItemPage({
       ? organizations[0]?.id?.toString()
       : null;
 
-  const toggleDescription = () => {
-    setIsExpanded(!isExpanded);
-  };
+  useEffect(() => {
+    async function fetchServiceCatalogItem() {
+      try {
+        const response = await fetch(
+          `/api/v2/help_center/service_catalog/items/${serviceCatalogItemId}`
+        );
+        const data = await response.json();
+        if (response.ok) {
+          setServiceCatalogItem(data.service_catalog_item);
+        }
```

Comment:
I also notice that we are not handling the case where the request is not successful. Shouldn't we have at least log the issue?

Reply:
I don't see this pattern in other places but we definitely will introduce some proper error handling with logging.